### PR TITLE
Feat css to scss

### PR DESCRIPTION
**Pull request by joanpadolina**
Updated our css to scss

### CODE REVIEW

File: public/css/scss/main.scss [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+@import "modules/colors.scss";
+// partials
+@import "partials/nav.scss";
+@import "partials/footer.scss";
+
+// animation
+@import "partials/loadingstate.scss";
+
+// pages
+@import "./modules/all.scss";
```
Comment (nlvo) [inline public/css/scss/main.scss]: I believe you don't have to add .scss at the end, nor ./ but only the / should be fine 🙂.

File: public/css/scss/modules/all.scss [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    margin: 0;
+    padding: 0;
+    background: var(--grey-200);
+}
+
+* {
+    box-sizing: border-box;
+}
+
+@import "../pages/index.scss";
```
Comment (nlvo) [inline public/css/scss/modules/all.scss]: Just a personal preference, but I would have a layouts/base or in your situation a pages/base where I would include everything from pages, rather then having them hidden in the modules folder. This way you wouldn't have to do something like '../pages/index'. Example from another project [marvel comics](https://github.com/nlvo/progressive-web-apps-1920/tree/master/src/scss)

`
@import 'characters';
@import 'comics';
@import 'browse';
@import 'search';
@import 'offline';
`

File: public/css/scss/main.scss (hunk 1 above)
Comment (joanpadolina) [reply public/css/scss/main.scss]: can you give an example

File: public/css/scss/modules/all.scss (hunk 2 above)
Comment (joanpadolina) [reply public/css/scss/modules/all.scss]: does this work if you don't name you file with an underscore?

File: public/css/scss/main.scss (hunk 1 above)
Comment (nlvo) [reply public/css/scss/main.scss]: well line 13 would be, `@import '/modules/all'` I believe.

File: public/css/scss/modules/all.scss (hunk 2 above)
Comment (nlvo) [reply public/css/scss/modules/all.scss]: I do think so, yes.

Comment (joanpadolina) [reply public/css/scss/modules/all.scss]: it didn't, renamed files with underscore

Comment (nlvo) [reply public/css/scss/modules/all.scss]: ah my bad, I thought it was used for things like partials.

**Review by nlvo — APPROVED** [review]

**Pull request merged**